__tests__/fixtures/install/install-should-skip-incompatible-optional-sub-dep/.npmrc

```diff
@@ -0,0 +1 @@
+yarn-offline-mirror=./mirror-for-offline
```

Comment:
Do you need offline mirror and .tgz files for the test?
I think you could be fine just using `file:` dependencies for tests